src/ManagedServices/ManagedServices/Commands/RemoveAzureRmManagedServcicesAssignment.cs

```diff
@@ -0,0 +1,93 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Extensions;
+using Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using System;
+using System.Management.Automation;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Commands
+{
+    [Cmdlet(
+        VerbsCommon.Remove,
+        Microsoft.Azure.Commands.ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ManagedServicesAssignment",
+        DefaultParameterSetName = DefaultParameterSet,
+        SupportsShouldProcess = true), OutputType(typeof(PSRegistrationDefinition))]
+    public class RemoveAzureRmManagedServcicesAssignment : ManagedServicesCmdletBase
+    {
+        protected const string DefaultParameterSet = "Default";
+        protected const string ByResourceIdParameterSet = "ByResourceId";
+        protected const string ByInputObjectParameterSet = "ByInputObject";
+
+        [Parameter(Position = 0, ParameterSetName = DefaultParameterSet, Mandatory = false, HelpMessage = "The scope where the registration assignment should be created.")]
+        [ScopeCompleter]
+        public string Scope { get; set; }
```

Comment:
What does scope do here if it is not mandatory?  Why would the user want to provide a scope?

Reply:
If the Scope is not provided, we default to /subs/id but assignment could have been created under /subs/id/rgs/myRG. If Scope is not provided as a parameter we will never to able to delete the assignment.